### PR TITLE
Fix the Critical and High vulnerabilities in cockroach-self-signer-cert

### LINKED ISSUE #316: critical and high vulnerabilities reported for cockroachdb/cockroach-self-signer-cert:v1.3
the following vulnerabilities were identified for the image [cockroachdb/cockroach-self-signer-cert:v1.3](https://console.cloud.google.com/gcr/images/cockroachlabs-helm-charts/global/cockroach-self-signer-cert@sha256:e225fe7eaa552fa85ff34ec2b5cb4b9a6e98db317d6f013390af336183f290c1/details)

Critical vulnerability:
[CVE-2022-40674](https://nvd.nist.gov/vuln/detail/CVE-2022-40674)
High-severity vulnerabilities:
[CVE-2023-0767](https://exchange.xforce.ibmcloud.com/vulnerabilities/247260)
[CVE-2022-37434](https://nvd.nist.gov/vuln/detail/CVE-2022-37434)
[CVE-2023-0286](https://nvd.nist.gov/vuln/detail/CVE-2023-0286)
[CVE-2022-42898](https://nvd.nist.gov/vuln/detail/CVE-2022-42898)

### PR DESCRIPTION
**Pull request by prafull01**
Fixes #316 #195 

Ran trivy scanner on image built with this code, it shows 0 CRITICAL and 0 HIGH vulnerabilities.

Screenshot 2023-06-06 at 12 50 08 PM

### CODE REVIEW

File: cockroachdb/templates/job.init.yaml [hunk 1]
```diff
@@ -122,16 +122,17 @@ spec:
...
                   local exitCode="$?";
                   echo $output;
 
-                  if [[ "$exitCode" == "0" || "$output" == *"cluster has already been initialized"* ]]
-                    then break;
+                  if [[ "$output" =~ .*"Cluster successfully initialized".* || "$output" =~ .*"cluster has already been initialized".* ]]; then
+                    break;
                   fi
 
+                  echo "Sleeping for 5 seconds"
```
Comment (rail) [inline cockroachdb/templates/job.init.yaml:135]: Can you add comments why we need to `sleep` here please.

**Review by rail — APPROVED** [review]
Thank you very much!

File: cockroachdb/templates/job.init.yaml (hunk 1 above)
Comment (prafull01) [reply cockroachdb/templates/job.init.yaml]: Added reason in the echo statement directly.

**Review by pseudomuto — APPROVED** [review]
Nice! LGTM 👍

**Pull request merged**